chainer/functions/math/bias.py

```diff
@@ -0,0 +1,29 @@
+from chainer.functions.array import broadcast
+from chainer.functions.array import reshape
+
+
+def bias(x, y, axis=1):
+    """Elementwise summation with broadcasting.
+
+    Computes a elementwise summation of two input variables, with the shape of
+    the latter variable broadcasted to match the shape of the former. `axis`
+    is the first axis of the first variable along which to apply the second
+    variable.
+
+    Args:
+        x (~chainer.Variable): Input variable to be scaled.
+        y (~chainer.Variable): Input variable to scale, broadcasted.
+        axis (int): The first axis of `x` along which to apply `y`
+
+    Returns:
+        ~chainer.Variable: Output variable.
+
+    """
+    x_shape = x.data.shape
+    y_shape = y.data.shape
+    assert x_shape[axis:axis + len(y_shape)] == y_shape
```

Comment:
Should enable this assertion only when debug mode? We can switch the mode with `chainer.set_debug` method.